src/test/java/io/confluent/idesidecar/restapi/kafkarest/RecordsV3ErrorsSuite.java

```diff
@@ -274,6 +275,8 @@ default void shouldThrowNotImplementedForUnsupportedSchemaDetails(ProduceRequest
   }
 
   @Test
+  // TODO: Figure out why this test fails for cp-demo
+  @DisabledIfSystemProperty(named = "running-in-cp-test-environment", matches = "true")
```

Comment:
Do we want to capture it in an issue?

Reply:
Created https://github.com/confluentinc/ide-sidecar/issues/236